sub-packages/bionemo-esm2/src/bionemo/esm2/scripts/train_esm2.py

```diff
@@ -346,6 +356,7 @@ def train_esm2_entrypoint():
         experiment_name=args.experiment_name,
         resume_if_exists=args.resume_if_exists,
         nemo1_init_path=args.nemo1_init_path,
+        create_checkpoint_callback=args.create_checkpoint_callback,
```

Comment:
Same applies:

Would you please mirror these changes in `sub-packages/bionemo-llm/src/bionemo/llm/train.py`